crates/wac-graph/src/encoding.rs

```diff
@@ -99,7 +99,7 @@ pub struct Scope {
     /// The map of types to their encoded indexes.
     pub type_indexes: IndexMap<Type, u32>,
     /// The map of imported instances in this scope.
```

Comment:
While we're here, this comment can probably be improved to indicate that the map is from instance name to instance index.